tests/sys/vane/clay.hoon

```diff
@@ -0,0 +1,208 @@
+/+  tester
+::
+/=  clay-raw  /:  /===/sys/vane/clay  /!noun/
+::
+!:
+=,  format
+::
+=/  test-pit=vase  !>(..zuse)
+=/  clay-gate  (clay-raw test-pit)
+::
+|_  _tester:tester
+++  test-info  ^-  tang
+  ::
+  =^  results1  clay-gate
+    %-  clay-call-with-comparator  :*
+      clay-gate
+      now=~1111.1.1
+      scry=*sley
+      ^=  call-args
+        :+  duct=~[/info]  type=-:!>(*task:able:clay)
+        ^-  task:able:clay
+        :^  %info  ~nul  %home
+        ^-  nori:clay
+        :-  %&
+        ^-  soba:clay
+        :~  [/file1/noun `miso:clay`[%ins [%noun %noun 'file1']]]
+            [/file2/noun `miso:clay`[%ins [%noun %noun 'file2']]]
+        ==
+      ^=  move-comparator
+        |=  moves=(list move:clay-gate)
+        ^-  tang
+        ::
+        ?.  ?=([* * * ~] moves)
+          [%leaf "wrong number of moves: {<(lent moves)>}"]~
+        ::
+        ^-  tang
+        ;:  welp
+          %-  expect-eq  !>
+          :_  i.moves
+          ^-  move:clay-gate
+          :-  duct=~[/info]
+          ^-  (wind note:clay-gate gift:able:clay)
+          :+  %pass  /castifying/~nul/home/~1111.1.1
+          ^-  note:clay-gate
+          :-  %f
+          [%build ~nul live=%.n [%pin ~1111.1.1 [%list ~]]]
+        ::
+          %-  expect-eq  !>
+          :_  i.t.moves
+          ^-  move:clay-gate
+          :-  duct=~[/info]
+          ^-  (wind note:clay-gate gift:able:clay)
+          :+  %pass  /diffing/~nul/home/~1111.1.1
+          ^-  note:clay-gate
+          :-  %f
+          [%build ~nul live=%.n [%pin ~1111.1.1 [%list ~]]]
+        ::
+          ^-  tang
+          ::
+          =/  move=move:clay-gate  i.t.t.moves
+          =/  =duct                                      p.move
+          =/  card=(wind note:clay-gate gift:able:clay)  q.move
+          ::
+          ^-  tang
+          %+  welp
+            (expect-eq !>([~[/info] duct]))
+          ::
+          ?.  ?=(%pass -.card)
+            [%leaf "bad move, not a %pass: {<move>}"]~
+          ::
+          =/  =wire  p.card
+          ::
+          ^-  tang
+          %+  welp
+            (expect-eq !>([/inserting/~nul/home/~1111.1.1 wire]))
+          ::
+          =/  note=note:clay-gate  q.card
+          ::
+          ?.  ?=([%f %build *] note)
+            [%leaf "bad move, not a %build: {<move>}"]~
+          ::
+          ^-  tang
```

Comment:
The only real thing that jumps out is the repeated `^-  tang` lines, which seem superfluous.